react/src/components/atoms/forms/Input/index.js

```diff
@@ -58,32 +115,35 @@ class InputProvider extends React.Component {
       // extra re-renders are avoided when context updates.
       if (!Object.prototype.hasOwnProperty.call(formContext.value, this.props.id)) {
         const { value } = formContext;
-        value[this.props.id] = { getValue: this.getValue, setValue: this.setValue };
+        value[this.props.id] = {
+          getValue: this.getValue,
+          setValue: this.setValue,
+          getLinkedContent: this.getLinkedContent,
+          setLinkedContent: this.setLinkedContent,
+          getUpdateFunc: this.getUpdateFunc,
+          setUpdateFunc: this.setUpdateFunc,
```

Comment:
I think using the exact names for these functions is confusing to me.. e.g. setUpdateFunc for each input vs from formContext

Reply:
Maybe in formContext can be named `setInputUpdateFunc`?